plugins/orchestrator-backend/src/service/router.ts

```diff
@@ -119,6 +119,15 @@ export async function createBackendRouter(
   router.use(permissionsIntegrationRouter);
   router.use('/workflows', express.text());
   router.use('/static', express.static(resolvePackagePath(pkg.name, 'static')));
+  router.use(
+    '/docs',
+    express.static(
+      resolvePackagePath(
+        '@janus-idp/backstage-plugin-orchestrator-common',
+        'src/generated/docs',
```

Comment:
once the issues with rebase have been resolved, i think this should be `docs/markdown`

Reply:
it can not be markdown. we expose the whole docs folder

Reply:
i made it docs/html